internal/planner/configuration.go

```diff
@@ -181,3 +199,18 @@ func hasShare(cfg smbcc.ConfigSection, k smbcc.Key) bool {
 	}
 	return false
 }
+
+func removeShare(cfg *smbcc.ConfigSection, k smbcc.Key) bool {
```

Comment:
You are using here pass-by-pointer `cfg *smbcc.ConfigSection`; for consistency, you should do so also for `hasShare` above (kind of slipped my eyes in previous review).

Reply:
Usually, when possible I like to pass by value on purpose to have unshared data. The hasShare function is "read only" whereas removeShare mutates the config section. So it's "intentionally" inconsitent. In this case it's totally fine to change it for consistency, but there may be cases where I push back more.  I'll try to note those in comments in the future.

Reply:
ACK